docs/eventing/samples/kafka/channel/README.md

```diff
@@ -1,156 +1,190 @@
-
-## Configuring the Knative broker for Apache Kafka channels
-
-1. To setup a broker that will use the new default Kafka channels, you must
-   create a new _default_ broker, using the command:
-   ```bash
-   kubectl create -f - <<EOF
-   apiVersion: eventing.knative.dev/v1
-   kind: Broker
-   metadata:
-    name: default
-   EOF
-   ```
+## Creating a Kafka Channel
+
+- Create a Kafka Channel that contains the following YAML:
+
+    ```yaml
+    apiVersion: messaging.knative.dev/v1beta1
+    kind: KafkaChannel
+    metadata:
+      name: my-kafka-channel
+      spec:
+        numPartitions: 3
+        replicationFactor: 1
+    ```
+
+## Specifying Kafka as the default Channel implementation
+<!--TODO: Move to admin guide-->
+
+1. To configure Kafka Channel as the [default channel configuration](../../../channels/channel-types-defaults), modify the `default-ch-webhook` ConfigMap so that it contains the following YAML:
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: default-ch-webhook
+      namespace: knative-eventing
+    data:
+      # Configuration for defaulting channels that do not specify CRD implementations.
+      default-ch-config: |
+        clusterDefault:
+          apiVersion: messaging.knative.dev/v1beta1
+          kind: KafkaChannel
+          spec:
+            numPartitions: 3
+            replicationFactor: 1
+    ```
+
+1. If Kafka is configured as the default Channel implementation for your cluster, creating a Channel with the following default YAML creates a Kafka Channel:
+
+    ```yaml
+    apiVersion: messaging.knative.dev/v1
+    kind: Channel
+    metadata:
+      name: testchannel-one
+    ```
+
+1. Verify that the Channel was created properly by checking that your Kafka cluster has a `testchannel-one` Topic. If you are using Strimzi, you can run the command:
+
+    ```bash
+    kubectl -n kafka exec -it my-cluster-kafka-0 -- bin/kafka-topics.sh --bootstrap-server my-cluster-kafka-bootstrap:9092 --list
+    ```
+
+    The output looks similar to the following:
+
+    ```
+    ...
+    __consumer_offsets
+    knative-messaging-kafka.default.my-kafka-channel
+    knative-messaging-kafka.default.testchannel-one
+    ...
+    ```
+
+    The Kafka Topic that is created by the Channel contains the name of the namespace, `default` in this example, followed by the name of the Channel. In the consolidated Channel implementation, it is also prefixed with `knative-messaging-kafka` to indicate that it is a Kafka Channel from Knative.
+
+    !!! note
+        The topic of a Kafka Channel is an implementation detail and records from it should not be consumed from different applications.
+
+## Configuring the Knative Broker for use with Apache Kafka Channels
+
+The following example uses a ApiServerSource to publish events to the Broker, and a Trigger that routes events to a Knative Service.
+<!--TODO: Not sure this example makes sense, why would you have an event source AND channels?-->
```

Comment:
@devguyio this doesn't really look Kafka related at all. I feel like we should remove it and write a proper procedure for what is trying to be achieved by this one instead? I'm confused by the use of event source etc here.

Reply:
I'll defer to @devguyio as I don't have a lot of context on use cases. From a strictly completeness point of view, I can see the value in an example that shows `Configuring the Knative Broker for use with Apache Kafka Channels`, but the example is not working in its current state and requires some TLC.

Reply:
I don't want to keep blocking the PR on this since it's just for general clean up, let's come back to this